concent_api/core/utils.py

```diff
@@ -121,14 +121,19 @@ def extract_name_from_scene_file_path(absoulte_scene_file_path_in_docker: str) -
     return relative_scene_file_path_in_archive
 
 
+def get_major_and_minor_golem_messages_version(protocol_version: str) -> str:
+    divided_version = protocol_version.split('.')
```

Comment:
I don't get this part - you are splitting the string at `.` just to join it in the next line.

Reply:
@rwrzesien Becuase sometimes we can get here 2 elements out of 3 and we want to store this version as a string in this form: `major_number.minor_number`. Golem Messages version has also patch which is unnecessary for us so I decided to do this convertion this way :)
@dybi I had a lot of thinking about good name for this function(what you wrote was one of it) and I still didn't like any of my thought about name :P If you want me to change name of this function I will. Also maybe today you got any new idea about the name? :D